flink-table/flink-sql-parser/src/main/codegen/includes/parserImpls.ftl

```diff
@@ -177,6 +177,70 @@ SqlDescribeDatabase SqlDescribeDatabase() :
 
 }
 
+SqlCreate SqlCreateFunction(Span s, boolean replace) :
+{
+    SqlIdentifier functionName = null;
```

Comment:
minor: similar to the other PR, rename to `functionIdentifier`?

Reply:
Same above.